telegram/_callbackquery.py

```diff
@@ -590,7 +592,7 @@ async def get_game_high_scores(
         connect_timeout: ODVInput[float] = DEFAULT_NONE,
         pool_timeout: ODVInput[float] = DEFAULT_NONE,
         api_kwargs: JSONDict = None,
-    ) -> List["GameHighScore"]:
+    ) -> Tuple["GameHighScore"]:
```

Comment:
```suggestion
    ) -> Tuple["GameHighScore", ...]:
```

Reply:
I really have no idea why mypy doesn't catch that :/ In this MWE mypy complains as desired:

```python
from typing import Tuple


class Bot:
    async def stuff(self) -> Tuple[str, ...]:
        return "a", "b", "c"


class Message:
    def __init__(self, bot: Bot):
        self.bot = bot

    async def other_stuff(self) -> Tuple["str"]:  
        return await self.bot.stuff()
```

Anyway, I regexed this error and hopefully got everything.